src/components/editor/index.js

```diff
@@ -120,38 +131,59 @@ class Draw extends Component {
     this.setState({showActions: false});
   };
 
+  _onToggleLayer = layer => {
+    const {selectedFeature} = this.props.editor;
+    if (!selectedFeature) {
+      return;
+    }
+
+    this.props.onTogglePolygonFilter(layer, selectedFeature.id);
+  };
+
   render() {
     const {
       className,
       clickRadius,
       datasets,
       editor,
-      features,
       layers,
+      filters,
+      onUpdate,
       style
     } = this.props;
+
     const {selectedFeature = {}} = editor;
+    const {lastPosition, showActions} = this.state;
+    const selectedFeatureId = (selectedFeature || {}).id;
+    const currentFilter = filters.find(f => f.value.id === selectedFeatureId);
+    const availableLayers = layers.filter(editorLayerFilter);
 
     return (
-      <StyledWrapper editor={editor} className={classnames('editor', className)} style={style}>
+      <StyledWrapper
+        editor={editor}
+        className={classnames('editor', className)}
+        style={style}
+      >
         <Editor
```

Comment:
<img width="460" alt="Screen Shot 2019-12-24 at 5 46 19 AM" src="https://user-images.githubusercontent.com/3605556/71381966-dac69100-2610-11ea-9ac6-e03b9a26b3fa.png">

Can you create a task in upgrade react-map-gl-drawer so this doesn't showup?

Reply:
Done: https://github.com/keplergl/kepler.gl/issues/877